sdk/communication/communication-identity/samples/v1/javascript/revokeTokens.js

```diff
@@ -16,11 +16,10 @@ dotenv.config();
 const connectionString =
   process.env["COMMUNICATION_CONNECTION_STRING"] || "<communication service connection string>";
 
-async function main() {
-  console.log("\n== Issue Token Javascript Sample ==\n");
+export const main = async () => {
+  console.log("\n== Issue Token Typescript Sample ==\n");
```

Comment:
This should be revoke, right?

Reply:
Also, maybe we should change the language to not be specific to TS because this gets compiled into TS and JS. Or call it TypeScript/JavaScript

Reply:
Right, Updated.